modules/@apostrophecms/area/ui/apos/components/AposAreaWidget.vue

```diff
@@ -615,11 +605,8 @@ export default {
     align-items: center;
     margin: 0;
     padding: 2px;
-    background-color: var(--a-primary);
-  }
-
-  .apos-area-widget-inner .apos-area-widget-inner .apos-area-widget__breadcrumbs {
-    background-color: var(--a-secondary);
+    background-color: var(--a-background-primary);
+    border: 1px solid var(--a-primary-button-hover);
```

Comment:
@abea i know this un-semantic use of variables will grind your gears, I suggest a ticket that converts them to something more agnostic of use, more like the base vars

Reply:
Is this not a find all and replace quick thing? Why not do it in this change?

Reply:
because it seemed like a different thing that would grind your gears

Reply:
i can do it that way though, sure

Reply:
Yeah, it'd be better to get it done. No idea when the ticket would happen.